### PR TITLE
feat: made landing page

### PR DESCRIPTION
**Pull request by Divya70**
made landing page using html and css

e-com-home
e-comhome-footer

### CODE REVIEW

File: index.css [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+      font-size: 1.5rem;
+}
+
+.login-btn {
+      background-color: var(--secondary-color);
+      border: none;
+      box-shadow: 7px 10px 12px -7px rgba(0, 0, 0, 0.75);
+      -webkit-box-shadow: 7px 10px 12px -7px rgba(0, 0, 0, 0.75);
+      -moz-box-shadow: 7px 10px 12px -7px rgba(0, 0, 0, 0.75);
+}
```
Comment (AnkurChunekar) [inline index.css:92]: One suggestion here, you can use a variable for reusability (remember DRY principle).

File: index.css [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+.explore-btn {
+      margin: 2rem;
+      padding: 0.8rem;
+      font-size: 1.2rem;
+      border: none;
+      background-color: var(--primary-color);
+      box-shadow: 7px 10px 12px -7px rgba(0, 0, 0, 0.75);
+      -webkit-box-shadow: 7px 10px 12px -7px rgba(0, 0, 0, 0.75);
+      -moz-box-shadow: 7px 10px 12px -7px rgba(0, 0, 0, 0.75);
+}
```
Comment (AnkurChunekar) [inline index.css:137]: here too, it can save time and helps to be consistent.

**Comment by AnkurChunekar** [comment]
Hey Great work divya, your homepage image has some unnecessary details like numbers, if this is you final image you can crop it a bit to display only necessary details.
Rest everything is great.

**Review by AnkurChunekar — APPROVED** [review]

**Review by Ayushi20-19 — APPROVED** [review]
few changes suggested by Ankur else good to go

**Pull request merged**